dashboard/src/main/webapp/src/api/api.js

```diff
@@ -16,63 +16,67 @@
-export function getShufflegetStatusTotal(params){
-    return http.get('/server/nodes/summary', params,{})
+export function getShufflegetStatusTotal(params,headers) {
+    return http.get('/server/nodes/summary', params, headers, 0)
 }
 
 // Create an interface for activeNodes
-export function getShuffleActiveNodes(params){
-    return http.get('/server/nodes?status=active', params,{})
+export function getShuffleActiveNodes(params,headers) {
+    return http.get('/server/nodes?status=active', params, headers, 0)
 }
 
 // Create an interface for lostNodes
-export function getShuffleLostList(params){
-    return http.get('/server/nodes?status=lost', params,{})
+export function getShuffleLostList(params,headers) {
+    return http.get('/server/nodes?status=lost', params, headers, 0)
 }
 
 // Create an interface for unhealthyNodes
-export function getShuffleUnhealthyList(params){
-    return http.get('/server/nodes?status=unhealthy', params,{})
+export function getShuffleUnhealthyList(params,headers) {
+    return http.get('/server/nodes?status=unhealthy', params, headers, 0)
 }
 
 // Create an interface for decommissioningNodes
-export function getShuffleDecommissioningList(params){
-    return http.get('/server/nodes?status=decommissioning', params,{})
+export function getShuffleDecommissioningList(params,headers) {
+    return http.get('/server/nodes?status=decommissioning', params, headers, 0)
 }
 
 // Create an interface for decommissionedNodes
-export function getShuffleDecommissionedList(params){
-    return http.get('/server/nodes?status=decommissioned', params,{})
+export function getShuffleDecommissionedList(params,headers) {
+    return http.get('/server/nodes?status=decommissioned', params, headers, 0)
 }
 
 // Create an interface for excludeNodes
-export function getShuffleExcludeNodes(params){
-    return http.get('/server/nodes?status=excluded', params,{})
+export function getShuffleExcludeNodes(params,headers) {
+    return http.get('/server/nodes?status=excluded', params, headers, 0)
 }
 
 // Total number of interfaces for new App
-export function getAppTotal(params){
-    return http.get('/app/total', params,{})
+export function getAppTotal(params,headers) {
+    return http.get('/app/total', params, headers, 0)
 }
 
 // Create an interface for the app basic information list
-export function getApplicationInfoList(params){
-    return http.get('/app/appInfos', params,{})
+export function getApplicationInfoList(params,headers) {
+    return http.get('/app/appInfos', params, headers, 0)
 }
 
 // Create an interface for the number of apps for a user
-export function getTotalForUser(params){
-    return http.get('/app/userTotal', params,{})
+export function getTotalForUser(params,headers) {
+    return http.get('/app/userTotal', params, headers, 0)
+}
+
+// Obtain the configured coordinator server list
+export function getAllCoordinatorAddrees(params) {
+    return http.get('/coordinatorList', params, {}, 1)
```

Comment:
Has this interface been committed?

Reply:
> Has this interface been committed?

This is the new interface. Get the drop-down list.